src/main/interface/settings.c

```diff
@@ -319,8 +320,15 @@ static const char * const lookupOverclock[] = {
     };
 #endif
 
+#ifdef USB_CDC_HID
+static const char* lookupUSB[] = {
+		"DEFAULT",
+		"COMPOSITE_HID"
```

Comment:
Can you see more modes being added here? If not, renaming the setting to  `usb_hid` and using OFF / on will be more efficient in terms of flash space.

Reply:
I'll move to ON, OFF for now. In case it changes in the future we will add it back again.